khard/helpers/__init__.py

```diff
@@ -61,6 +62,101 @@ def get_random_uid() -> str:
     return ''.join([random.choice(string.ascii_lowercase + string.digits)
                     for _ in range(36)])
 
+def yaml_clean(value: Union[str, List]) -> Optional[Union[List[str], str]]:
+    """
+    sanitize yaml values according to some simple principles:
+      1. empty values are none, so ruamel does not print an empty list/str
+      2. list with only one item become this item
+      3. multiline strings use the YAML literal style:
+         https://yaml.org/spec/1.2.2/#literal-style
+    
+    :param value: the value to be sanitized
+    :returns: the sanitized value
+    """
+    # special case for empty values
+    if not value:
+        value = None
```

Comment:
You do not need the local variable `value` (or rather overwrite the argument), you can just `return` directly.

Also see https://pheanex.github.io/pylint/